### DISCUSSION TITLE
Best way to install the CLI on M1 mac with no internet from on-prem nexus repo with

### DISCUSSION THREAD

**Discussion by zhanhanamazon** [aws/aws-cli#7778] [Q&A] [answered]
Hi, 

how to install the AWSCLIv2 from a on-prem nexus repo to which my m1 mac laptop connect. 
The laptop and the repo do not have internet connection and they are all in an enterprise network.

**Comment by tim-finnigan** [comment, marked as answer]
Here is documentation for installing the AWS CLI: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html. You'll need an internet connection to download the installation files required for your OS. You can also build and install from source as documented here: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-source-install.html.
